escoot/de-at/augment-gemeinsamer-service-und-mietvertrag-de-at.md

```diff
@@ -206,7 +202,7 @@ Wenn die Erfüllung des **Vertrags** durch höhere Gewalt verhindert oder verzö
 
 **Augment** haftet nicht für indirekte Schäden oder Folgeschäden, außer wenn **Augment** diese Schäden oder Folgeschäden absichtlich oder durch grobe Fahrlässigkeit verursacht hat.
 
-<div class="page"></div>
+<div class="page"/>
```

Comment:
Page breaks should follow [special markup](https://github.com/augment-eco/terms-and-conditions?tab=readme-ov-file#special-markup).

```suggestion
<div class="page"></div>
```